torch/csrc/autograd/profiler_python.cpp

```diff
@@ -249,6 +256,28 @@ class ValueCache {
 
   void trimPrefixes();
 
+  template <CallType C>
+  typename Config<C>::cls_t set_class(
```

Comment:
This feels strange as part of ValueCache. It seems like there should be a base class for the cache of these specializations.

Reply:
updated as a standalone function